.circleci/docker/common/install_rocm.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -ex
+
+install_ubuntu() {
+    apt-get update
+    apt-get install -y wget
+    apt-get install -y libopenblas-dev
+
+    # Need the libc++1 and libc++abi1 libraries to allow torch._C to load at runtime
+    apt-get install libc++1
+    apt-get install libc++abi1
+
```

Comment:
Can we add libsndfile1 for ubuntu and libsndfile for centos so that it can removed from .jenkins/pytorch/test.sh.
Tried to do that in PR#34465, but its not required now.

Reply:
I'll add that to the follow-up PR which will also do the apt-cache cleanup.